controllers/cloud.redhat.com/providers/web/impl.go

```diff
@@ -249,11 +245,8 @@ func makeKeycloak(o obj.ClowdObject, objMap providers.ObjectMap, usePVC bool, no
 		FailureThreshold:    3,
 	}
 
-	image := fmt.Sprintf("quay.io/keycloak/keycloak:%s", KEYCLOAK_VERSION)
-
-	if clowderconfig.LoadedConfig.Images.Keycloak != "" {
-		image = clowderconfig.LoadedConfig.Images.Keycloak
-	}
+	env := o.(*crd.ClowdEnvironment)
```

Comment:
This type assert could fail, and it will panic if it does. As the code stands a) it shouldn't and b) MakeCachedComponent doesn't take an error for the `fn` argument so I can understand why you don't try to return an error here. We should fix this SOON!